utils/ext_auth.py

```diff
@@ -18,7 +19,25 @@ def setup_external_auth_ipa(**data):
         iparealm: Realm.
         credentials: Key of the credential in credentials.yaml
     """
+    connect_kwargs = {
+        'username': credentials['host_default']['username'],
+        'password': credentials['host_default']['password'],
+        'hostname': data['ipaserver'],
+    }
+    import fauxfactory
+    appliance_name = 'cfmeappliance' + (fauxfactory.gen_alpha(7)).lower()
+    appliance_address = appliance.IPAppliance().address
+    appliance_fqdn = appliance_name + '.' + data['iparealm'].lower()
+    ssh = SSHClient(**connect_kwargs)
```

Comment:
Ahhh that's what I thought - maybe make this clearer by the name of the variable